README.md

```diff
@@ -100,6 +101,22 @@ Parameters:
 
 You should lock your deployments to a specific version tag. We guarantee that these tags will be immutable- once they are released the will not change.
 
+##### Amazon ECR Public Gallery
+
+[aws-for-fluent-bit](https://gallery.ecr.aws/aws-observability/aws-for-fluent-bit)
+
+Our images are available in Amazon ECR Public Gallery. We recommend our customers to find images with different tags here by following command:
+
+```
+docker pull public.ecr.aws/aws-observability/aws-for-fluent-bit:<tag>
+```
+
+For example, you can get the image with latest version by:
+
+```
+docker pull public.ecr.aws/aws-observability/aws-for-fluent-bit:latest
+```
```

Comment:
Let's add a comment recommending that they log into ECR public using AWS CLI so that they don't face download limits

and then link to hte ECR public docs.

On the amazon network I always have to log into ecr public with AWS CLI to pull. Others on corporate VPNs may face that too